src/Components/Api.js

```diff
@@ -43,4 +43,10 @@ export const deleteComment =(commentId)=>{
   return ncApi.delete(`/comments/${commentId}`).then((res)=>{
     return;
   })
-}
\ No newline at end of file
+}
+
+export const getArticlesByTopic = (slug) => {
+  return ncApi.get(`/articles/?topic=${slug}`).then((res) => {
```

Comment:
Would encourage you to use axios query params here https://masteringjs.io/tutorials/axios/get-query-params